main.go

```diff
@@ -578,11 +578,25 @@ func transformCompile(args []string) ([]string, error) {
 	return append(flags, newPaths...), nil
 }
 
+var blacklistedPackages = [...]string{
+	// Hardcoded variables and methods names
+	"runtime",
+
+	// Constants in asm code
+	"internal/cpu",
+	"internal/bytealg",
+}
+
 // isPrivate checks if GOPRIVATE matches path.
 //
 // To allow using garble without GOPRIVATE for standalone main packages, it will
 // default to not matching standard library packages.
 func isPrivate(path string) bool {
+	for _, blacklistedPackage := range blacklistedPackages {
+		if path == blacklistedPackage || strings.HasPrefix(path, blacklistedPackage+"/") {
+			return false
+		}
+	}
```

Comment:
You can simplify this further; you could simply add `,runtime,internal/cpu,internal/bytealg` to the existing GOPRIVATE after we've worked it out. The existing GOPRIVATE algorithm already does this "exact match or path prefix" logic.